### DISCUSSION TITLE
Is there anyway to add multiple bindings to same action for mini.comment

### DISCUSSION THREAD

**Discussion by neonvoidx** [nvim-mini/mini.nvim#649] [Q&A] [answered]
Currently using the default bindings for mini.comment, but on top of having `gc` to comment a line, I also want to add a bind to use `Ctrl + /` to do the same thing

**Comment by echasnovski** [comment, marked as answer]
Sorry, 'mini.comment' is not designed for this. But you can use one thing that should work (and I think will work in the foreseeable future).

Call `setup()` twice with different mappings: one for `gc` and the other for `<C-/>`. As `setup()` only creates mappings and does not delete, this should work as expected. So something like this:
```lua
require('mini.comment').setup()
require('mini.comment').setup({
  mappings = {
    comment = '<C-/>',
    comment_line = '<C-/><C-/>',
    comment_visual = '<C-/>',
  },
})
```

Note that I tried that in my terminal emulator and it could not use `<C-/>` per se (although mappings are there). `<M-/>` worked, though.

**Reply by neonvoidx** [reply]
thank you
